docs/excel/excel-add-ins-events.md

```diff
@@ -36,7 +36,10 @@ Any change that complies with default behavior of Excel will trigger the corresp
 
 ### Lifecycle of an event handler
 
-An event handler is created when an add-in registers the event handler and is destroyed when the add-in unregisters the event handler or when the add-in is closed. Event handlers do not persist as part of the Excel file.
+An event handler is created when an add-in registers the event handler. It is destroyed when the add-in unregisters the event handler or when the add-in is refreshed, reloaded, or closed. Event handlers do not persist as part of the Excel file, or across sessions with Excel Online.
+
+> [!CAUTION]
+> When the object to which events are registered is deleted (e.g. a table with an onChanged event), the event handler is not cleaned up. It no longer triggers but remains in memory until the add-in or Excel session refreshes or closes.
```

Comment:
Suggest revising the note to something more like this:

When an object to which events are registered is deleted (e.g., a table with an `onChanged` event registered), the event handler no longer triggers but remains in memory until the add-in or Excel session is refreshed or closed.

- formatted `onChanged` as code
- added comma after "e.g."
- minor rewording